lib/evm_database_ops.rb

```diff
@@ -119,15 +117,40 @@ def self.restore(db_opts, connect_opts = {})
         backup_folder = action == :dump ? "db_dump" : "db_backup"
         uri = File.join(connect_opts[:uri], backup_folder, connect_opts[:remote_file_name])
       end
+    else
+      uri = db_opts[:local_file]
+
+      # HACK(ish): Since everything is flowing through MiqFileStorage so that
+      # we can handle the file splitting logic all in one place, this means
+      # that local dumps/backups also need to now have a "uri" in the
+      # connection opts so that MiqLocalMountSession will be used.
+      #
+      # This just puts the bare minimum necessary for URI.parse to recognize
+      # the :uri option as "file" scheme, and allows MiqFileStorage to then
+      # instanciate MiqLocalMountSession below in the `.with_interface_class`
```

Comment:
typo `s/instanciate/instantiate/`

Reply:
Would you believe me if I even made sure to [`:Spellcheck`](https://github.com/NickLaMuro/dotfiles/blob/master/.vimrc.d/spellcheck.vim) prior to pushing... and still didn't catch this...